### ISSUE TITLE
ios, v14.6 webgl1 no Texture is shown

### ISSUE THREAD

**Issue by arpu** [protectwise/troika#199]
Hello, testing on ios 14.6 with wegl1 on Browserstack and get no Texture shown 

ios > 15.0 with wegl1 works just fine on same device on BrowserStack 

i cannot get any more infos or logs right now 

![Bildschirmfoto von 2022-03-25 18-36-15](https://user-images.githubusercontent.com/14173/160172852-e1003b0b-2919-4664-bc10-e3184cd00592.png)
![Bildschirmfoto von 2022-03-25 18-36-09](https://user-images.githubusercontent.com/14173/160172855-bd131ce8-e667-4422-b6f3-572ab4bdc99e.png)

    "troika-three-text": "^0.46.3",
    "troika-three-utils": "^0.46.0",

**Comment by arpu** [comment]
tested with troika example on https://troika-examples.netlify.app/#text

same result in the console i get 

![Bildschirmfoto von 2022-03-28 13-40-06](https://user-images.githubusercontent.com/14173/160390415-1d650ab1-ea22-45d9-b19b-0d6754bd374b.png)

**Comment by lojjic** [comment]
Hmm, thanks.
Does it also fail with [gpuAccelerateSDF:false](https://protectwise.github.io/troika/troika-three-text/#gpuacceleratesdf)? 
Does it work with 0.45?
I wish I had a real iOS 14 device to test with, just to make sure this isn't some browserstack quirk.

**Comment by arpu** [comment]
can test this tomorrow!

BrowserStack should be a real IOS14 device not?

**Comment by arpu** [comment]
have done some short testing using 
 downgrade to:
```
    "troika-three-text": "^0.45.0",
    "troika-three-utils": "^0.45.0",
```
works!

setting [gpuAccelerateSDF:false](https://protectwise.github.io/troika/troika-three-text/#gpuacceleratesdf) 
on 0.46 does not help

**Comment by lojjic** [comment]
OK, thank you for checking those. It sounds like it may have an issue with using a secondary webgl canvas as a texture. I'll try to debug this in browserstack soon.

**Comment by lojjic** [comment]
Made some progress -- if I `readPixels` out of the sdf canvas and put them in a DataTexture then iOS14 is able to display it. Still don't quite understand the root cause, but worst case I could do that as a workaround on <=iOS14 only and it'll just take a slight performace hit.

**Comment by lojjic** [comment]
@arpu I've published that workaround to 0.46.4, can you verify that fixes it on your app?

For the record, this bug also appeared on Safari <=14 on MacOS, it's not limited to iOS.

**Comment by arpu** [comment]
does the https://troika-examples.netlify.app/#text use the new build  0.46.4 ?

**Comment by lojjic** [comment]
No that's the `main` branch, I'll need to port the fix back there but wanted to verify the 0.46 branch first.

This one is 0.46: https://deploy-preview-201--troika-examples.netlify.app/#text
